The check_include function in the setup script passes $CC as argv[0] to
subprocess.call, which fails in our cross environment where $CC includes some
extra command-line flags. Just assume the headers are found.

--- a/setup.py
+++ b/setup.py
@@ -89,9 +89,6 @@
     )
 
 
-if not check_include('python3', 'Python.h'):
-    sys.exit(2)
-
 module_poopt = Extension('poezio.poopt',
                     extra_compile_args=['-Wno-declaration-after-statement'],
                     sources=['poezio/pooptmodule.c'])
